is-game-over.js

```diff
@@ -0,0 +1,13 @@
+"use strict"
+const _ = require("lodash");
+
+function isGameOver(gameState) {
```

Comment:
This function will return true if the current player has won. But I think it would be more useful to tell if either player has won. That way, if the UI is loading a game state from storage, it can call this function to tell if it should go to the "game over" state or the "keep playing state", regardless of who moved last.